### ISSUE TITLE
Regex issue

### ISSUE THREAD

**Issue by jbboehr** [Rich-Harris/tippex#15]
I'm trying to compile handlebars.js using rollup and rollup-plugin-typescript, and it seems to be choking on this:
https://github.com/wycats/handlebars.js/blob/v4.0.5/lib/handlebars/compiler/compiler.js#L429-L431

```
Error transforming /home/rin/Code/launchpad/handlebars.js/handlebars.js/lib/handlebars/compiler/compiler.js: rollup-plugin-typescript: Unexpected character (439:3). If this is valid JavaScript, it's probably a bug in tippex. Please raise an issue at https://github.com/Rich-Harris/tippex/issues – thanks!


  },

    ^; when processing: 'handlebars.js/lib/handlebars/compiler/compiler.js'
Error: Error transforming handlebars.js/lib/handlebars/compiler/compiler.js: rollup-plugin-typescript: Unexpected character (439:3). If this is valid JavaScript, it's probably a bug in tippex. Please raise an issue at https://github.com/Rich-Harris/tippex/issues – thanks!


  },

    ^; when processing: '/home/rin/Code/launchpad/handlebars.js/handlebars.js/lib/handlebars/compiler/compiler.js'
    at getErasedCode (/usr/lib/nodejs/rollup-plugin-typescript/dist/rollup-plugin-typescript.cjs.js:144:9)
    at fix (/usr/lib/nodejs/rollup-plugin-typescript/dist/rollup-plugin-typescript.cjs.js:113:18)
    at Object.transform (/usr/lib/nodejs/rollup-plugin-typescript/dist/rollup-plugin-typescript.cjs.js:271:50)
    at /usr/lib/nodejs/rollup/src/utils/transform.js:19:35
    at process._tickCallback (node.js:368:9)
    at Function.Module.runMain (module.js:444:11)
    at startup (node.js:136:18)
    at node.js:966:3
```

**Comment by jbboehr** [comment]
Actually, I'm not sure it's related to the `$`, but it compiles when I remove those three regexes.

**Comment by jlowcs** [comment]
I have a similar issue. By elimination, I concluded that the cluprit is the `\/` at the end of the regex. If I remove it, rollup doesn't fail anymore.

Until it is resolved, you can use `[/]` instead:

```javascript
/foo\//
/foo[/]/
```

**Comment by Rich-Harris** [comment]
Thanks! Fixed in 2.3.1
